src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,45 @@
+package lotto.domain;
+
+public enum Rank {
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 30_000_000),
+    THIRD(5, 1_500_000),
+    FOURTH(4, 50_000),
+    FIFTH(3, 5_000),
+    MISS(0, 0);
+
+    private int countOfMatch;
+    private int winningMoney;
```

Comment:
```suggestion
    private final int countOfMatch;
    private final int winningMoney;
```

변하지 않는 값에는 `final` 키워드를 붙여 주면 좋겠죠~